### ISSUE TITLE
Transport_Services

### ISSUE THREAD

**Issue by AlexKrashevsky** [ChrisHird/intERPrise#13]
I was able to run the build process in my system (CRTOBJS).

Now wanted to put the service program SVRFUNC to test.

README file mentions program and command TESTSVR. There are SVRTEST program and command instead. Is this a README typo? 

Tried to run SVRTEST command. Ran with default parameters, it returned error: "Unknown host".

When typing in the IP address of the IBM i server for first parameter, the process returns the following error:

Failed to connect to socket : A remote host did not respond within the
  timeout period.                                                     

It appears that I am failing at connect() API. What may I be missing?

**Comment by ChrisHird** [comment]
Alex
Yes typo for the program name. Can only enter the IP address for test but will address resolution at later time. The program does wait on a key stroke from the user. It allows me to carry out other tests ( still new stuff being developed).
Did you make sure the server and worker jobs are running? Looks like the request to connect is waiting for something to happen?
Debug is the same as debug for RPG just set break point where you want to see what’s going on. Server debug is a bit more difficult. I am running before code upload so should work on yours unless the config is not correct.

Happy to teamviewer if needed.

Chris

More will become clear when I release next update.

Sent from my iPhone

________________________________

**Comment by AlexKrashevsky** [comment]
Chris - thanks.

Started the subsystem via IRPTOPS menu, still timing out on connect().

**Comment by ChrisHird** [comment]
Any messages or joblogs I can review please attach

Sent from my iPhone

________________________________

**Comment by AlexKrashevsky** [comment]
[QPJOBLOG_ALEXK_QPADEV0005_021292_1.pdf](https://github.com/ChrisHird/intERPrise/files/2622777/QPJOBLOG_ALEXK_QPADEV0005_021292_1.pdf)

Job Log is attached.

**Comment by ChrisHird** [comment]
Alex

The server is not running. Will look at tomorrow as late. Will improve the set up
Instructions

Sent from my iPhone

________________________________

**Comment by AlexKrashevsky** [comment]
Sounds good, thanks for looking into this, later.

**Comment by ChrisHird** [comment]
Jobs that should be running in the subsystem (option 5 OPSMENU)
IRPTSBS        QSYS        SBS      .0                   DEQW 
  IRPSVR       CHRISH      BCH      .0  PGM-IRP0000      DEQW 
  IRP0001      CHRISH      BCI      .0  PGM-IRP0001      TIMW 
  IRP0001      CHRISH      BCI      .0  PGM-IRP0001      TIMW 
  IRP0001      CHRISH      BCI      .0  PGM-IRP0001      TIMW 
  IRP0001      CHRISH      BCI      .0  PGM-IRP0001      TIMW 
  IRP0001      CHRISH      BCI      .0  PGM-IRP0001      TIMW 

Server config

Server Listening Port  . . . . . . . . .   23456       
Number of Server Jobs  . . . . . . . . .   5           
Session Time out Interval  . . . . . . .   10          

NETSTAT shows listening ports
*                *          23456      013:43:04  Listen 
look at jobs using connection (option 8)
                         Display Jobs Using Connection                        
                                                             System:   SAS4   
 Connection type  . . . . . . :   *TCP                                        
 Local address  . . . . . . . :   *                                           
 Local port . . . . . . . . . :   23456                                       
 Remote address . . . . . . . :   *                                           
 Remote port  . . . . . . . . :   *                                           
                                                                              
 Type options, press Enter.                                                   
   5=Work with job                                                            
                                                                              
                                                            Current           
 Opt     Name           User           Number     Type      User              
         IRPSVR         CHRISH         131403     *BCH      CHRISH            
         IRP0001        CHRISH         131404     *BCH      CHRISH            
         IRP0001        CHRISH         131405     *BCH      CHRISH            
         IRP0001        CHRISH         131406     *BCH      CHRISH            
         IRP0001        CHRISH         131407     *BCH      CHRISH            
         IRP0001        CHRISH         131408     *BCH      CHRISH            
Interfaces 
     127.0.0.1        127.0.0.0        *LOOPBACK    Active   
     192.168.200.93   192.168.200.0    ETHLINE      Active   

Make sure the IP address is correct (*LOCALHOST does not work)  (ping). 
my connection request =
SVRTEST ADDR('192.168.200.93') PRF(CHRISH) PWD()

Let me know what you have missing?

**Comment by AlexKrashevsky** [comment]
I'll retry tonight and report back to the issue.

Thanks.

**Comment by ChrisHird** [comment]
Issue closed as superseded by code updates.
